### DISCUSSION TITLE
custom loss function

### DISCUSSION THREAD

**Discussion by arnon-1** [kohya-ss/sd-scripts#811] [Q&A]
Hello,

I'd like to try training on images using a custom loss function for training a lora model. In particular, I want to be able to emphasise training on certain parts of images.
I do not request this to be implemented, but I would appreciate a hint of what loss function I should mess with (in the code) for a local quick and dirty solution.

Thanks in advance!

**Comment by araleza** [comment]
The upcoming 'masked loss' feature should provide what you need:

https://github.com/kohya-ss/sd-scripts/pull/589

**Reply by arnon-1** [reply]
Ah nice, I don't even need to code anything myself :-)
